examples/combineConfigs.md

```diff
@@ -92,9 +92,9 @@ secret holding them (not covering that here).
 <!--
 secretGenerator:
 - name: app-tls
-  commands:
-    tls.crt: "cat tls.cert"
-    tls.key: "cat tls.key"
+  literals:
+    tls.crt=./tls.cert
+    tls.key=./tls.key
```

Comment:
is this example correct? it looks like it would create secret keys with the contents `./tls.cert`, etc, which probably isn't what someone would want

Reply:
My bad, will change those literals to better examples.